drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -829,6 +830,10 @@ def slFiles2dataTables(slNames):
                           "FLUX2": "Flux(0.125)", "MAGNITUDE1": "MagAp(0.03)", "MAGNITUDE2": "MagAp(0.125)",
                           "MERR1": "MagErr(0.03)", "MERR2": "MagErr(0.125)", "MSKY": "MSky(0.125)",
                           "STDEV": "Stdev(0.125)", "FLAGS": "Flags", "ID": "ID", "CI": "CI"}
+    titleSwapDict_dao4 = {"X": "X-Center", "Y": "Y-Center", "RA": "RA", "DEC": "DEC", "FLUX1": "n/a",
+                          "FLUX2": "FluxAp2", "MAGNITUDE1": "MagAp1", "MAGNITUDE2": "MagAp2",
+                          "MERR1": "MagErrAp1", "MERR2": "MagErrAp2", "MSKY": "MSkyAp2",
+                          "STDEV": "StdevAp2", "FLAGS": "Flags", "ID": "ID", "CI": "CI"}
     titleSwapDict_daoTemp={"X":"XCENTER","Y":"YCENTER","RA":"n/a","DEC":"n/a","FLUX1":"FLUX1","FLUX2":"FLUX2","MAGNITUDE1":"MAG1","MAGNITUDE2":"MAG2","FLAGS":"n/a","ID":"ID","MERR1":"MERR1","MERR2":"MERR2","MSKY":"MSKY","STDEV":"STDEV"}
     titleSwapDict_sourceX={"X":"X_IMAGE","Y":"Y_IMAGE","RA":"RA","DEC":"DEC","FLUX1":"FLUX_APER1","FLUX2":"FLUX_APER2","MAGNITUDE1":"MAG_APER1","MAGNITUDE2":"MAG_APER2","FLAGS":"FLAGS","ID":"NUMBER"}
```

Comment:
If this is the only mapping of the SEGMENT columns, then this will not work.  Do you want to update or shall I in another issue?

Reply:
updated in commit [ba8fc3b](https://github.com/spacetelescope/drizzlepac/pull/481/commits/ba8fc3b83fa5cd233cdf8e60b33cd4e6744d0508)